src/track/external-captions-handler.js

```diff
@@ -240,7 +240,7 @@ class ExternalCaptionsHandler extends FakeEventTarget {
   reset(externalTracks: Array<TextTrack>): void {
     this._resetCurrentTrack();
     this._textTrackModel = {};
-    this._setPrefixForExternalNativeTextTrack(externalTracks);
+    this._deleteExternalNativeTextTrack(externalTracks);
```

Comment:
delete is not the best name I think as we are not deleting it not does the HTML5 spec exposes a delete method.
```suggestion
    this._resetExternalNativeTextTrack(externalTracks);
```